src/test/kotlin/com/karumi/maxibonkata/DevelopersTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.karumi.maxibonkata
+
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.properties.forAll
+import io.kotlintest.specs.ShouldSpec
+
+class DevelopersTest : ShouldSpec() {
+    init {
+
+        "Developers" {
+            should("always grab a positive number of maxibons") {
```

Comment:
Is zero a positive number? 🤔 😄

Reply:
It depends on your definition of positive 😈

Reply:
I used to make a difference between Natural and Positive numbers?
* Positive number: [1, +∞)
* Natural number: [0, +∞)

I know there is no consense about that. But [Ada95](http://www.adahome.com/rm95/), the best language ever does this distinction :P http://www.adaic.org/resources/add_content/standards/05rm/html/RM-3-5-4.html#I1677